### PR TITLE
Disable loki and promtail configuration

### PR DESCRIPTION
**Pull request by Simranpal**
With this PR, the `loki` and `promtail` configuration is disabled. It is because the package is not available from official SUSE repositories and package.

Basically, commented the parts related to `loki` and `promtail`. I have tested with deployment.

### CODE REVIEW

File: salt/monitoring_srv/init.sls [hunk 1]
```diff
@@ -1,4 +1,3 @@
 include:
   - .prometheus
   - .grafana
-  - .loki
```
Comment (arbulu89) [inline salt/monitoring_srv/init.sls:4]: I would comment this instead of removing

File: salt/monitoring_srv/loki.sls [hunk 2]
```diff
@@ -1,3 +1,4 @@
+{# loki package is currently disabled. To use it, add the .loki sls entry in init.sls and enable loki as datasource in grafana/datasources template #}
```
Comment (arbulu89) [inline salt/monitoring_srv/loki.sls:1]: If you comment/remove the `loki` entry in the init.sls file, we don't really need to comment this

**Review by arbulu89 — APPROVED** [review]
It looks good. Just some minor comments

File: salt/monitoring_srv/init.sls (hunk 1 above)
Comment (MalloZup) [reply salt/monitoring_srv/init.sls:4]: yes

**Review by MalloZup — CHANGES_REQUESTED** [review]
I'm not sure it will work this because we are missing the client part.

https://github.com/SUSE/ha-sap-terraform-deployments/blob/16b27530f03552448d3610b598a8602e8b04f747/salt/cluster_node/monitoring.sls#L30

**Comment by Simranpal** [comment]
closing this as too many conflict issues, I have created https://github.com/SUSE/ha-sap-terraform-deployments/pull/579 for `sap-blue-horizon` branch as discussed

**Pull request closed without merging**